src/components/shared/UserAvatar/UserAvatar.tsx

```diff
@@ -25,33 +22,30 @@ interface Props extends Pick<AvatarProps, 'size' | 'className' | 'notSet'> {
   showLink?: boolean;
   /** The corresponding user object if available */
   user?: User | null;
-
+  /** Use the user's thumbnail instead of full-size avatar */
   preferThumbnail?: boolean;
 }
 
 const displayName = 'UserAvatar';
 
 const UserAvatar = ({
```

Comment:
I don't know if it makes sense anymore but I always wanted to get rid of the `notSet` prop since it was added as a crutch the first time around.

Maybe not necessarily now, but at least when we redo the UI we definitely have to get rid of it